crystal-map-connector/src/main/kotlin/ch/ergon/dope/CrystalMapAdapter.kt

```diff
@@ -15,7 +15,7 @@ import com.schwarz.crystalapi.schema.CMObjectList
 import com.schwarz.crystalapi.schema.CMType
 import com.schwarz.crystalapi.schema.Schema
 
-fun CMType.asField(reference: String = path): Field<out ValidType> = Field(
+fun CMType.toDopeField(reference: String = path): Field<out ValidType> = Field(
```

Comment:
Rename all this functions also to `toDopeType`.

Reply:
done